### PR TITLE
[scatter] support country

### PR DESCRIPTION
**Pull request by chejennifer**
- allow placetype of country to be selected and when country is selected, show place as Earth
- programmed together with Bo

dev updated (go to https://dev.datacommons.org/tools/scatter and choose country from the place type dropdown)

### CODE REVIEW

**Review by beets — APPROVED** [review]
very cool!
would also be nice if we could search for "earth" in the search box, and default the selection to country. maybe file a feature request?

File: static/js/tools/scatter/app.tsx [hunk 1]
```diff
@@ -115,10 +115,7 @@ function shouldDisplaySpinner(isLoading: IsLoadingWrapper): boolean {
  */
 function shouldShowChart(x: Axis, y: Axis, place: PlaceInfo): boolean {
   return (
-    !_.isEmpty(place.enclosedPlaceType) &&
-    !_.isEmpty(place.enclosingPlace.dcid) &&
-    !_.isNull(x.statVarInfo) &&
-    !_.isNull(y.statVarInfo)
+    isPlacePicked(place) && !_.isNull(x.statVarInfo) && !_.isNull(y.statVarInfo)
```
Comment (shifucun) [inline static/js/tools/scatter/app.tsx:118]: can add a function "isStatVarPicked"?

File: static/js/tools/scatter/util.ts [hunk 2]
```diff
@@ -252,6 +252,26 @@ function updateHashDisplayOptions(
...
+    (!_.isEmpty(place.enclosedPlaceType) &&
+      !_.isEmpty(place.enclosingPlace.dcid))
+  );
+}
+
+/**
+ * Checks if the child places have been loaded.
+ * @param place
+ */
+export function arePlacesLoaded(place: PlaceInfo): boolean {
```
Comment (shifucun) [inline static/js/tools/scatter/util.ts:271]: this function seems not used anymore..

**Review by shifucun — COMMENTED** [review]
Thanks for making this working!

File: static/js/tools/scatter/app.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/scatter/app.tsx]: done

File: static/js/tools/scatter/util.ts (hunk 2 above)
Comment (chejennifer) [reply static/js/tools/scatter/util.ts:271]: it's still used in chart_loader actually

**Comment by shifucun** [comment]
@googlebot I consent.

**Review by shifucun — APPROVED** [review]

**Pull request merged**